.chloggen/debug-exporter-default-verbosity-basic.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Since the change is essentially a no-op, would this really be a breaking change?

Reply:
Hmm yeah maybe not. I'll change it to enhancement then.